src/Tag.h

```diff
@@ -67,7 +67,7 @@ namespace OP2Utility
 
 
 	// Factory method to construct a Tag from a string literal
-	inline constexpr Tag MakeTag(const char(&tagText)[5]) {
+	constexpr Tag MakeTag(const char(&tagText)[5]) {
 		return Tag(tagText);
 	}
```

Comment:
Note for posterity: This use of `inline` is outside of a class, though because of the `constexpr` it is still implicitly `inline`.

https://en.cppreference.com/w/cpp/language/inline
> A function declared constexpr is implicitly an inline function.

----

All the other methods with `inline` being removed are defined within a class definition:
> A function defined entirely inside a class/struct/union definition, whether it's a member function or a non-member friend function, is implicitly an inline function.

Reply:
Thank you for reviewing.